sdk/spring/spring-messaging-azure-servicebus/src/main/java/com/azure/spring/messaging/servicebus/core/DefaultServiceBusNamespaceProducerFactory.java

```diff
@@ -107,7 +107,6 @@ private ServiceBusSenderAsyncClient doCreateProducer(String name, @Nullable Prod
             ProducerProperties producerProperties = parentMerger.merge(properties, this.namespaceProperties);
             producerProperties.setEntityName(entityName);
 
-            //TODO(yiliu6): whether to make the producer client share the same service bus client builder
```

Comment:
We should track this in the issue.

Reply:
Good point, could you please help create the issues?

Reply:
created: https://github.com/Azure/azure-sdk-for-java/issues/28313